src/client/components/SupplierAddressEditor/AddressCountry.react.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import request from 'superagent-bluebird-promise';
+
+/**
+ * Supplier address list table
+ *
+ * @author Dmitry Divin
+ */
+class AddressCountry extends Component {
```

Comment:
I put it here because it returns a html `<td></td>` tag which I don't think it's a generic use case to be extracted to a component